schema/rpc.fbs

```diff
@@ -102,7 +103,6 @@ table SteamVRTrackersSetting {
     feet: bool;
     knees: bool;
     elbows: bool;
-    hands: bool;
```

Comment:
Are you sure this is supposed to be removed? @Louka3000 what do you think?

Reply:
That was a mistake